src/app/content/reducer.ts

```diff
@@ -82,7 +82,7 @@ function reduceContent(state: State, action: AnyAction) {
       }
 
       // book is the same, page is different
-      if (state.book && state.page && action.payload.match.params.page !== getPageSlug(state.book, state.page)) {
+      if (state.book && state.page && action.payload.match.params.page.slug !== getPageSlug(state.book, state.page)) {
```

Comment:
eventually we're gonna make it so you can have different page arguments than just slug, you can future proof by comparing `state.params.page` and `action.payload.match.params.page` with isEqual. incidentally, the comparison up on line 75 can be simplified to avoid the `omitPage` just as you did in `resolveContent`